web/src/ProviderEditPage.js

```diff
@@ -256,6 +256,79 @@ class ProviderEditPage extends React.Component {
             </React.Fragment>
           )
         }
+        {
+          this.state.provider.type !== "Custom" ? null : (
+            <React.Fragment>
+              <Row style={{marginTop: '20px'}} >
+                <Col style={{marginTop: '5px'}} span={(Setting.isMobile()) ? 22 : 2}>
+                  {Setting.getLabel(i18next.t("provider:Auth URL"), i18next.t("provider:Auth URL - Tooltip"))}
```

Comment:
Can you provide example values for these CustomXXX params? You can directly use: https://door.casdoor.com/ values. https://door.casdoor.com/ will support to be connected. We use real examples.

Reply:
In commit c021fd834c6ccaf620e575a77d08cd6b509bb11e, I set some example values for customAuthUrl, customScope, customTokenUrl and customUserInfoUrl